src/Core/Components/NavMenu/FluentNavLink.razor.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.Fast.Components.FluentUI.Utilities;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+public partial class FluentNavLink : FluentNavBase
+{
+    private readonly RenderFragment _renderContent;
+
+    internal string? ClassValue => new CssBuilder("fluent-nav-item")
```

Comment:
You always need to create the CssBuilder using the user Class: `new CssBuilder(Class).AddClass("fluent-nav-item")`.
To ensure the correct order of optional overloads.